### ISSUE TITLE
is this plugin still supported 

### ISSUE THREAD

**Issue by mayersgamer** [abnerfs/cs2-rockthevote#72]
any way the vote is not getting players attention , i think you should add like a game freeze option when vote starts , most players dont vote unlike the offcial game vote.

**Comment by abnerfs** [comment]
Yes, just paused development for a while because I'm really busy with work, I don't think I understood your proposal, like pausing the game so the vote happens or something like that?
